### PR TITLE
Neethu

### PR DESCRIPTION
**Pull request by Neethu-Philip**
Included circleci pipeline configuration for building helm repo

### CODE REVIEW

File: .circleci/build.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    echo "REPO_URL found empty. Exiting ..."
+    exit 1
+  fi
+}
+
+echo "performing sanity checks ..."
+sanity_check
+
+echo "initializing helm ..."
+helm init --client-only
```
Comment (sami-alajrami) [inline .circleci/build.sh]: this is a helm v2 command. The dockerfile uses helm v3.2.1

File: .circleci/build.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    exit 1
+  fi
+
+  if [ -z "${CYBERDOJO_HELM_REPO_NAME}" ]; then
+    echo "REPO_NAME found empty. Exiting ..."
+    exit 1
+  fi
+
+  if [ -z "${CYBERDOJO_HELM_REPO_BUCKET_NAME}" ]; then
+    echo "REPO_URL found empty. Exiting ..."
```
Comment (sami-alajrami) [inline .circleci/build.sh]: This is REPO_BUCKET_NAME

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+    build:
+      docker:
+        - image: praqma/helm-aws #Change to helm-gcs
+      steps:
+        - checkout
+        - run: chmod +x .circleci/build.sh
```
Comment (sami-alajrami) [inline .circleci/config.yml]: you can do the `chmod +x .circleci/build.sh` on your machine and push it. Then you can remove this line from the pipeline config

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+    build:
+      docker:
+        - image: praqma/helm-aws #Change to helm-gcs
```
Comment (sami-alajrami) [inline .circleci/config.yml]: change this before merging

File: .circleci/dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# adapted from https://github.com/hypertrace/helm-gcs-packager/blob/master/Dockerfile
+# manually built and pushed as praqma/helm-gsutil
+FROM google/cloud-sdk:alpine
+
+# Install Helm
+RUN curl -Lo helm-v3.1.2-linux-amd64.tar.gz https://get.helm.sh/helm-v3.1.2-linux-amd64.tar.gz && \
```
Comment (sami-alajrami) [inline .circleci/dockerfile]: this installs helm v3.1.2 , but the script uses helm v2.X commands. You can either update the script to use helm v3 commands or install helm v2 here

**Review by sami-alajrami — CHANGES_REQUESTED** [review]

File: .circleci/build.sh (hunk 1 above)
Comment (Neethu-Philip) [reply .circleci/build.sh]: Docker file updated to use helm v2

Comment (Neethu-Philip) [reply .circleci/build.sh]: Updated docker file to use helm v2

File: .circleci/build.sh (hunk 2 above)
Comment (Neethu-Philip) [reply .circleci/build.sh]: Done

**Review by sami-alajrami — COMMENTED** [review]
you need to push the praqma/helm-gsutils docker image so that the pipeline can run on this PR and you see if it passes

**Review by sami-alajrami — APPROVED** [review]

**Pull request merged**